TipCatDotNet.Api/Services/HospitalityFacilities/AccountService.cs

```diff
@@ -30,7 +32,7 @@ public Task<Result<AccountResponse>> Add(MemberContext context, AccountRequest r
                     .Bind(AddDefaultFacility)
                     .Bind(AttachToMember)
                     .Tap(ClearCache)
-                    .Bind(accountId => GetAccount(accountId, cancellationToken)));
+                    .Bind(accountId => GetAccount(accountId, null, cancellationToken)));
```

Comment:
We added a default facility above, so here isn't null

Reply:
Yes, i'll try to figure out how i can store created account's id, because i need it for both bind methods. But my first binded method will return facilities and accountId will be lost.

Reply:
I found out how to solve it, but tests are crushing... Says like test can't find any facilities
Solution:
```
return Validate()
    .BindWithTransaction(_context, () => AddAccount()
        .Bind(AddDefaultFacility)
        .Bind(AttachToMember))
    .BindWithTransaction(_context, accountId => _facilityService.Get(context, accountId, cancellationToken)
        .Bind(facilities => GetAccount(accountId, facilities, cancellationToken))
        .Tap(ClearCache));
```

Reply:
Not perfect, but looks like a solution